packages/nimble-components/src/card/template.ts

```diff
@@ -1,4 +1,9 @@
 import { html } from '@microsoft/fast-element';
 import type { Card } from '.';
 
-export const template = html<Card>`<slot></slot>`;
+export const template = html<Card>`
+    <section aria-labelledby="titleSlot">
+        <slot name="title" id="titleSlot"></slot>
```

Comment:
nit for classes and ids prefer kebab case names, ie `title-slot`

Reply:
Fixed. The banner has `titleSlot` in camel case [here](https://github.com/ni/nimble/blob/f159a084527c7e53bac99dd43182c10c2f2f628f/packages/nimble-components/src/banner/template.ts#L53). I can update that in this PR, too, if you don't mind pulling in unrelated files.